apps/frontend/src/modules/projects/projects-api.ts

```diff
@@ -28,6 +32,23 @@ class ProjectApi extends BaseHTTPApi {
 
 		return await response.json<ProjectGetAllResponseDto>();
 	}
+
+	public async update(
```

Comment:
https://github.com/BinaryStudioAcademy/bsa-2024-gitfit/pull/107#discussion_r1738321776
same about patch namings (I used "patch" literally for everything)